docs/requirements.txt

```diff
@@ -5,7 +5,9 @@ alabaster_jupyterhub
 # 0.1.0 released.
 https://github.com/jupyterhub/autodoc-traits/archive/75885ee24636efbfebfceed1043459715049cd84.zip
 pydata-sphinx-theme
+pytablewriter==0.56.1
 recommonmark>=0.6
 sphinx-copybutton
 sphinx-jsonschema
+sphinx-markdown-tables==0.0.10
```

Comment:
I'm not sure about pinning these dependencies to the major.minor.patch version. We should at least not pin pytablewriter that has a minor version to its patch version.